modules/apps/frontend-taglib/frontend-taglib-sample-web/derived-config.txt

```diff
@@ -0,0 +1,25 @@
+com/liferay/frontend/taglib/sample/web/internal/display/context/SearchPaginatorDisplayContext.lambda$getSearchContainer$0(Ljava/util/List;[I)Ljava/util/List;:1
+com/liferay/frontend/taglib/sample/web/internal/display/context/SearchPaginatorDisplayContext.<init>(Lcom/liferay/portal/kernel/util/Portal;Ljavax/portlet/RenderRequest;Ljavax/portlet/RenderResponse;)V
+com/liferay/frontend/taglib/sample/web/internal/display/context/SearchPaginatorDisplayContext.getSearchContainer()Lcom/liferay/portal/kernel/dao/search/SearchContainer;:UNKNOWN
+org/apache/jsp/partials/search_005fpaginator_jsp.<clinit>()V
+org/apache/jsp/partials/search_005fpaginator_jsp.<init>()V
+org/apache/jsp/partials/search_005fpaginator_jsp.getDependants()Ljava/util/List;:SAFE
+org/apache/jsp/partials/search_005fpaginator_jsp._jspService(Ljavax/servlet/http/HttpServletRequest;Ljavax/servlet/http/HttpServletResponse;)V^0:0,UNKNOWN^1:1,UNKNOWN
+com/liferay/frontend/taglib/sample/web/constants/SamplePortletKeys.<init>()V
+com/liferay/frontend/taglib/sample/web/internal/portlet/SamplePortlet.<init>()V
+com/liferay/frontend/taglib/sample/web/internal/portlet/SamplePortlet.doDispatch(Ljavax/portlet/RenderRequest;Ljavax/portlet/RenderResponse;)V^0:0,UNKNOWN^1:1,UNKNOWN
```

Comment:
These look like some auto-generated junk artifacts, I don't know where they are coming from. This should not be committed.